core/src/main/java/com/dtolabs/rundeck/core/common/FrameworkResource.java

```diff
@@ -38,6 +33,8 @@
 public class FrameworkResource implements IFrameworkResource {
     public static final String VALID_RESOURCE_NAME_CHARSET_REGEX = "[-_a-zA-Z0-9+][-\\._a-zA-Z0-9+]*";
     public static final String VALID_RESOURCE_NAME_REGEX = "^"+VALID_RESOURCE_NAME_CHARSET_REGEX+"$";
+    public static final String VALID_RESOURCE_DESCRIPTION_CHARSET_REGEX = "^[a-zA-Z0-9\\s\\.,\\(\\)-]+$";
```

Comment:
```suggestion
    public static final String VALID_RESOURCE_DESCRIPTION_CHARSET_REGEX = "[a-zA-Z0-9\\s\\.,\\(\\)-]+";
```

The charset regex should not have `^` and `$`